app/main.ipynb

```diff
@@ -23,24 +20,29 @@
     "Also check for data correctness (salary & salary $K)."
```

Comment:
You forgot to check for data correctness (salary & salary $K).
If the values is correct